### ISSUE TITLE
Some config files are created under root

### ISSUE THREAD

**Issue by phahulin** [poanetwork/deployment-playbooks#110]
Some configuration files are owned by root, some are owned by corresponding role-user (`moc`, `bootnode`, etc). They all should probably be owned by role-user.

**Comment by ArseniiPetrovich** [comment]
These permissions are explicitly granted for files. @phahulin, are you sure about that they should be owned by <role> user? 
As from my point of view - yes, the processes that are using these files (for example `node.toml`) are launched from <role> user, so we should change the permissions accordingly. But i might have missed something.
It is easy to fix this issue, but we need to be sure, that the new approach is correct.
```
owner=root group=root mode=0644
```

**Comment by phahulin** [comment]
@ArseniiPetrovich Yes, let's change it and have them all owned by role-specific user.
